src/io.c

```diff
@@ -86,6 +89,7 @@ void fs_open(char *path, int rw)
         /* telling "floppy" for A:/B:, "ramdisk" for the rest */
     }
 #endif
+    dev_size = stbuf.st_size;
```

Comment:
Is this the size of a block device correctly?

Reply:
real block device does not return proper size value. so modify it to get device size by lseek()